packaging/nnstreamer.spec

```diff
@@ -206,6 +206,15 @@ ninja -C build %{?_smp_mflags}
 %install
 DESTDIR=%{buildroot} ninja -C build %{?_smp_mflags} install
 
+pushd %{buildroot}%{_libdir}
+ln -s %{gstlibdir}/libnnstreamer.so libnnstreamer.so
```

Comment:
Use -sf instead of -s in build script. Depending on toolchains, the current code may break the build if libnnstreamer.so does not exist at %{gstlibdir} at this point. Note that not-existing at %{gstlibdir} at this point is not abnormal.

Reply:
Fixed.